pkg/installer/custominstallconfig_test.go

```diff
@@ -14,10 +14,12 @@ import (
 	"github.com/Azure/azure-sdk-for-go/profiles/2018-03-01/resources/mgmt/resources"
 	"github.com/Azure/azure-sdk-for-go/profiles/latest/compute/mgmt/compute"
 	"github.com/Azure/go-autorest/autorest/to"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
```

Comment:
The validate job is complaining about this, but if you rebase it will be [fixed](https://github.com/openshift/installer-aro-wrapper/commit/d6af50321aced4a322a645539afa39420b679469#diff-2fb99f8c330ed4bb742aecc47f525262654eb8ac09d4f064790f965242a87ec8).